ui/app/src/App.tsx

```diff
@@ -94,7 +98,7 @@ function App() {
 
   // Fetch logs starting from this block
   // TODO: find a way to get the SM deployment block
-  const fromBlock = 4615000n;
+  const fromBlock = config.fromBlock ? BigInt(config.fromBlock) : undefined;
```

Comment:
Wouldn't null be better?

Reply:
The expected type in getContractEvents is `fromBlock?: bigint | BlockTag | undefined`, TS doesn't like it when i replace it with null

Reply:
oh, is that what `?` means? a type or `undefined`?

Reply:
yes that's why the '?' in the optional param; if you still pass an optional param when calling the function (as in our case), you'll need to pass it as undefined